src/main/java/mods/coww/libcd/CauldronRecipeTweaker.java

```diff
@@ -0,0 +1,33 @@
+package mods.coww.libcd;
+
+import io.github.cottonmc.libcd.api.tweaker.TweakerManager;
+import io.github.cottonmc.libcd.api.tweaker.recipe.RecipeParser;
+import io.github.cottonmc.libcd.api.tweaker.recipe.RecipeTweaker;
+import mods.coww.recipes.CauldronRecipe;
+import net.minecraft.item.ItemStack;
+import net.minecraft.recipe.Ingredient;
+import net.minecraft.util.DefaultedList;
+import net.minecraft.util.Identifier;
+
+public class CauldronRecipeTweaker {
+    public void addCauldronRecipe(Object[] inputs, String fluid, String catalyst, String output){
+        try {
+            ItemStack outputItemStack = RecipeParser.processItemStack(output);
```

Comment:
`output` should be an Object.

Reply:
doesn't that mean the catalyst should also be an object, since it's an ingredient

Reply:
oh wait right, the catalyst is an ingredient too, so yes